File: Tests/test6/devowel_file.pl
#!/usr/bin/perl

$file = $ARGV[0];

open FILE, '<', $file;

@final;
foreach $line (<FILE>) {
	@array = split("", $line);
	foreach $i (@array) {
		if ($i =~ /[aeiouAEIOU]/) {		
			next;
		}
		push(@final, $i);
	}
}
close FILE;

#/a.*e.*i.*o.*u/

open FILE, '>', $file;
print FILE @final;
close FILE;

File: Tests/test6/devowel_file.py
#!/usr/bin/python3

import re

f = open("frost.txt", "r")

data = f.readlines()

final = []
for line in data:
	line = re.sub(r"[aeiouAEIOU]","",line)
	final.append(line)

f.close()

f = open("frost.txt", "w")
f.close()

f = open("frost.txt", "w")
for line in final:
	f.write(line)
f.close()
